### DISCUSSION TITLE
Add support for BungeeCord

### DISCUSSION THREAD

**Discussion by mocha666** [BG-Software-LLC/SuperiorSkyblock2#612] [Approved Feature Requests] [answered]
Create different servers for different islands (load balancer, sync cache with Redis and more)
You can add support for bungee, which can be better combined with SWM

Example: We have three spigot servers as sub servers of skylock. Each sub server limits the number of empty Island worlds generated. After joining the server, players load the empty Island worlds in the sub database of enough free worlds in the world through load balancing. Players of each island can send them to the corresponding sub server when using the return Island command, After the player goes offline, the island world is saved to the database.

This will affect neighbour islands as islands would have to be loaded and pasted asyc on player join event. And removed again on player leave.

**Comment by mocha666** [comment]
well in the future a one island one world setup would be better.

https://spigotmc.org/resources/slimeworldmanager.69974
https://hypixel.net/threads/dev-blog-5-storing-your-skyblock-island.2190753/

**Comment by OmerBenGera** [comment, marked as answer]
You can track progress here:
<https://github.com/BG-Software-LLC/SSBBungeeBridge>
